core/templates/services/playthrough.service.ts

```diff
@@ -114,25 +116,35 @@ class CyclicStateTransitionsTracker {
 }
 
 class EarlyQuitTracker {
-  private stateName: string = null;
-  private expDurationInSecs: number = null;
+  private stateName: string = ' ';
```

Comment:
This should never be ' ' -- that is not a valid state name.

Reply:
What should i add in place of this ,i mean i tried to initialise it with everything but it just dont make sense then i added `" "` this but vojtechjelinek corrected me and then i modified it with ` ' ' ` .